package.json

```
@@ -19,8 +19,9 @@
   "scripts": {
     "fmt": "npm run --silent clean && deno fmt --ignore=node_modules/",
     "lint": "npm run --silent clean && deno lint --ignore=node_modules/",
-    "prepare": "deno run --unstable --allow-read --allow-write='lib/' --allow-env src/cli.ts src/tsconfig.json",
-    "clean": "git clean -fXde '!node_modules/'"
+    "prepare": "npm test",
+    "clean": "git clean -fXde '!node_modules/'",
+    "test": "deno run --unstable --allow-read --allow-write='lib/' --allow-env src/cli.ts src/tsconfig.json"
```

```suggestion
    "test": "npm run --silent prepare",
    "prepare": "deno run --unstable --allow-read --allow-write='lib/' --allow-env src/cli.ts src/tsconfig.json",
    "clean": "git clean -fXde !node_modules/"
```

Let me explain.

`prepare` shouldn't be an alias to `test`. `deno2node` might get unit tests one day.

However, I have no issue with an alias going the other way. Especially when I'm working on a CLI that runs Deno tests on Node.js. Like Node.js, it only runs `.[cm]?js`, making transpilation in `test` script necessary anyway.

<hr>

I like clean diffs. I'm annoyed when adding a comma is the only change to a line. I like dangling commas, but they're invalid in JSON.

So I put `clean` as the last script on purpose: I expect it to change very rarely. I'd rather it stayed there.

However, it could use a change right about now. Some recent-ish npm upgrade must've changed how quoting works, since the script was broken on `cmd`. I fixed it in another project by removing the apostrophes (and then changing my npm shell to bash anyway 🙂).